cmd/manager/main.go

```diff
@@ -62,7 +57,7 @@ func main() {
 	// implementing the logr.Logger interface. This logger will
 	// be propagated through the whole operator, generating
 	// uniform and structured logs.
-	logf.SetLogger(zap.Logger())
+	logf.SetLogger(logf.ZapLogger(true))
```

Comment:
`logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"` has been marked as deprecated https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/runtime/log/log.go, but I think we can take it by far, since it is part of the transition.